### PR TITLE
Automatically release a new version of flyctl daily

### PR DESCRIPTION
**Pull request by billyb2**
### Change Summary

What and Why:
By automatically releasing flyctl, as long as a PR gets approved and merged, it'll show up in a new flyctl release relatively soon. We can also release a new version if necessary.

How:
Added a new Github Action that runs every day at 3 PM eastern

Related to:
https://flyio.slack.com/archives/C06G424TJRF/p1718101085197859?thread_ts=1718047902.923779&cid=C06G424TJRF

---

### Documentation

- [ ] Fresh Produce
- [ ] In superfly/docs, or asked for help from docs team
- [x] n/a

### CODE REVIEW

File: .github/workflows/auto-release.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  run_script:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master branch
+        uses: actions/checkout@v3
+        with:
+          ref: master
+      - name: bump version
+        run: |
+          sh scripts/force_bump_version.sh
```
Comment (matthewlehner) [inline .github/workflows/auto-release.yml:17]: you should be able to run this without `sh` explicitly:

```suggestion
          ./scripts/force_bump_version.sh
```

**Review by matthewlehner — APPROVED** [review]
Looks good to me!

**Comment by billyb2** [comment]
Oh I realized, we should skip deploying on Fridays and weekends to not cause trouble when most of us are offline

**Comment by clouvet** [comment]
@billyb2 can also we get a note on the README that explains the release schedule and how it works?

**Review by clouvet — COMMENTED** [review]
@billyb2 can also we get a note on the README that explains the release schedule and how it works?

**Pull request merged**